romanisim/tests/test_l1.py

```diff
@@ -84,6 +84,39 @@ def test_apportion_counts_to_resultants():
     log.info('DMS229: successfully generated ramp from counts.')
 
 
+def test_linearized_counts_to_resultants():
+    """Test that we can apportion linearized counts to resultants.
+    """
+    counts = np.random.poisson(100, size=(100, 100))
+    coeffs = np.asfarray([1.0, 0.7, 3.0e-6, 5.0e-12])
+    lin_coeffs = np.tile(coeffs[:,np.newaxis,np.newaxis], (1, 100, 100))
+    lin_coeffs[:, 0:50, :] *= 2.0
+    gain = 4.0
+    counts[0,0] = counts[0,99] = counts[99,0] = counts[99,99] = 11.0
+
+    linearity = nonlinearity.NL(lin_coeffs, gain)
+
+    for tij in tijlist:
+        resultants, dq = l1.apportion_counts_to_resultants(counts.copy(), tij.copy())
```

Comment:
Do counts or tij get touched here?  Are these copys necessary?

Reply:
Code left in from debugging. `copy`s removed.